mono/utils/dlmalloc.h

```diff
@@ -28,6 +28,8 @@
 #include <stddef.h>   /* for size_t */
 #include <mono/utils/mono-compiler.h>
 
+#define NO_MALLINFO 1 /* mono */
```

Comment:
why?

(not saying it's wrong; just unclear why we don't need it)

Reply:
Yeah.. could remove or fix the code.
It is because returning an undefined struct by value is a warning or error and the code is apparently not used.

Reply:
Yeah.. could remove or fix the code.
It is because returning an undefined struct by value is a warning or error and the code is apparently not used.